packages/server/src/fhir/search.ts

```diff
@@ -344,8 +442,11 @@ async function getSearchIncludeEntries(
             value: canonicalReferences.join(','),
           },
         ],
-      })
-    );
+        count: DEFAULT_MAX_SEARCH_COUNT,
```

Comment:
Before this PR, since this search didn't specify count, `SelectQuery.limit_` had a value of `NaN` which resulted in no LIMIT clause being included. Perhaps that was the intention here, but it certainly feels like relying on `NaN` to achieve that is not ideal.